atomics/T1204.002/T1204.002.yaml

```diff
@@ -321,3 +321,71 @@ atomic_tests:
     cleanup_command: |
       Remove-Item #{pua_file}
 
+- name: Office Generic Payload Download
+  description: |
+    This Test uses a VBA macro to launch Powershell which will download a file from a user defined web server.
+    Required input agruments are c2_domain, c2_parent_directory, file_name
+    Execution is handled by [Invoke-MalDoc](https://github.com/redcanaryco/invoke-atomicredteam/blob/master/Public/Invoke-MalDoc.ps1) to load and execute VBA code into Excel or Word documents.
```

Comment:
can you update the link to point to Invoke-Maldoc in it's new location (like you have in the executor command)